data/human/message buoys.txt

```diff
@@ -0,0 +1,390 @@
+# Copyright (c) 2025 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+# Entering Republic space
+mission "Message Buoy: Entering Republic space from Pirates"
+	entering
+	repeat
+	destination "Earth"
+	invisible
+	to offer
+		has "chosen sides"
+		not "temporary ceasefire"
+		not "entered system by: takeoff"
+		has "previous system government: Pirate"
+	source
+		government "Republic"
+		neighbor government "Pirate"
```

Comment:
If your previous system was a pirate system and your current system is a Republic system, then you're necessarily in a system where `neighbor government "Pirate"` is true, so the `neighbor` filter is redundant. The same goes for the rest of the missions.